src/index.js

```diff
@@ -686,6 +687,19 @@ class Fastly {
   async updateBackend(version, name, data) {
     return this.request.put(`/service/${this.service_id}/version/${await this.getVersion(version, 'current')}/backend/${encodeURIComponent(name)}`, data);
   }
+
+  /**
+   * Create a new backend for a particular service and version.
+   *
+   * @see https://docs.fastly.com/api/config#backend_85c170418ee71191dbb3b5046aeb6c2c
+   * @param {number} version - The version number (current if omitted).
+   * @param {Object} data - The backend definition.
+   * @returns {Promise} The reponse object.
+   * @fulfil {Response}
+   */
+  async createBackend(version, data) {
+    return this.request.post(`/service/${this.service_id}/version/${await this.getVersion(version, 'current')}/backend`, data);
```

Comment:
I don't particular like the inline `await this.getVersion(...)` inside the template string...it also makes the line rather long. but I guess this is a matter of style :-)